src/client/helpers/equalizerAuxMethods.js

```diff
@@ -44,16 +44,20 @@ export const uploadSoundInfoFromFile = (eventFromInputFile, props) => {
   } = props;
   startCreationAudioDataAsProp && startCreationAudioDataAsProp();
   const [file] = eventFromInputFile.length ? eventFromInputFile : eventFromInputFile.target.files;
-  const audioFile = new Audio(URL.createObjectURL(file));
-  const sound = new Pizzicato.Sound({
-    source: 'file',
-    options: {
-      path: audioFile.src,
-      loop: true,
-    },
-  }, () => {
-    storeSound && removeSoundFilters(props);
-    !onToggle && attachFiltersToSource(sound, blocksData);
-    createSoundInfoInState(sound, file, props);
-  });
+  const fileName = file.name;
+  const ext = fileName.substring((fileName.lastIndexOf('.')) + 1);
```

Comment:
I would suggest to use more explicit names, because I personally don’t understand what ext is it

Reply:
@tonypntsrn done